package.json

```diff
@@ -32,11 +33,22 @@
     "@stardazed/streams": "^3.1.0",
     "@types/marked": "^2.0.3",
     "buffer": "^6.0.3",
+    "big.js": "^6.1.1",
```

Comment:
Do we need this? Ethers.js already comes with a BigNumber library.

Reply:
no we dont. I need to remove unnecessary packages also